db-engine/src/main/java/com/splicemachine/db/impl/sql/execute/UserDefinedAggregator.java

```diff
@@ -195,9 +206,18 @@ public void writeExternal(ObjectOutput out) throws IOException
 	public void readExternal(ObjectInput in) 
 		throws IOException, ClassNotFoundException
 	{
-		in.readInt();   // unused until we have a second rev of this class
-
-        _aggregator = (Aggregator) in.readObject();
+		int v = in.readInt();
```

Comment:
[minor] replace tabs with spaces

Reply:
This file has mixed indenting, both options would look bad. I'd rather fix this with a repository wide reformatting effort.